tests/YoTennis.Tests/Test/EnumerableExtensionsTests.cs

```diff
@@ -0,0 +1,195 @@
+            var result = sequence.WithoutElement("two");
+            result.Should().BeEquivalentTo(new[] { "one" });
+        }
+
+        [Fact]
+        public void Test_WithoutElement_Int()
+        {
+            var sequence = new[] { 1, 2 };
+            var result = sequence.WithoutElement(2);
+            result.Should().BeEquivalentTo(new[] { 1 });
+        }
+
+        [Fact]
+        public void Test_Null_WithElement_String()
+        {
+            IEnumerable<string> sequence = null;
+            var result = sequence.WithElement("one");
+            result.Should().BeEquivalentTo(new[] { "one" });
+        }
+
+        [Fact]
+        public void Test_Null_WithElement_Int()
+        {
+            IEnumerable<int> sequence = null;
+            var result = sequence.WithElement(1);
+            result.Should().BeEquivalentTo(new[] { 1 });
+        }
+
+        [Fact]
+        public void Test_Null_WithoutElement_String()
+        {
+            IEnumerable<string> sequence = null;
+            var result = sequence.WithoutElement("one");
+            result.Should().BeEquivalentTo(new string[] { });
+        }
+
+        [Fact]
+        public void Test_Null_WithoutElement_Int()
+        {
+            IEnumerable<int> sequence = null;
+            var result = sequence.WithoutElement(1);
+            result.Should().BeEquivalentTo(new int[] { });
+        }
+
+        [Fact]
+        public void Test_WithElement_String_WithNull()
+        {
+            var sequence = new[] { null, "two" };
+            var result = sequence.WithElement("three");
+            result.Should().BeEquivalentTo(new[] { null, "two", "three" });
+        }
+
+        [Fact]
+        public void Test_WithoutElement_String_WithNull_WithCompare()
+        {
+            var forCompare = StringComparer.OrdinalIgnoreCase;
+            var sequence = new[] { null, "two", "three" };
+            var result = sequence.WithoutElement("tWo", forCompare);
+            result.Should().BeEquivalentTo(new string[] { null, "three" });
+        }
+
+        [Fact]
+        public void Test_WithoutElement_String_WithNull_WithCompare2()
+        {
+            var forFirstCompare = new FirstEqualityComparer();
+            var sequence = new[] { null, "two", "three" };
+            var result = sequence.WithoutElement("tWo", forFirstCompare);
+            result.Should().BeEquivalentTo(new string[] { null });
+        }
+
+        [Fact]
+        public void Test_WithoutElement_String_WithCompare()
+        {
+            var forFirstCompare = new FirstEqualityComparer();
+            var sequence = new[] { "one", "two", "three" };
+            var result = sequence.WithoutElement("five", forFirstCompare);
+            result.Should().BeEquivalentTo(new[] { "one", "two", "three" });
+        }
+
+        [Fact]
+        public void Test_WithoutElement_Int_WithCompare3()
+        {
+            var IEComparer = new IntEqualityComparer();
```

Comment:
do not call local variables like this. local variables must be `camelCased`. `PascalCase` is used for classes, properties and so on. Moreover, format `IPascalCase` is meant to be used for interfaces. In your case just use `comparer` here.